pkg/webui/account/views/oauth-authorization-settings/index.js

```diff
@@ -0,0 +1,168 @@
+// Copyright © 2022 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react'
+import { connect } from 'react-redux'
+import { defineMessages, useIntl } from 'react-intl'
+import { Col, Row, Container } from 'react-grid-system'
+import { bindActionCreators } from 'redux'
+import { push } from 'connected-react-router'
+
+import toast from '@ttn-lw/components/toast'
+import Button from '@ttn-lw/components/button'
+import DataSheet from '@ttn-lw/components/data-sheet'
+import Tag from '@ttn-lw/components/tag'
+import TagGroup from '@ttn-lw/components/tag/group'
+
+import DateTime from '@ttn-lw/lib/components/date-time'
+
+import attachPromise from '@ttn-lw/lib/store/actions/attach-promise'
+import PropTypes from '@ttn-lw/lib/prop-types'
+import sharedMessages from '@ttn-lw/lib/shared-messages'
+
+import { deleteAuthorization } from '@account/store/actions/authorizations'
+
+import { selectSelectedAuthorization } from '@account/store/selectors/authorizations'
+
+const RIGHT_TAG_MAX_WIDTH = 140
+
+const m = defineMessages({
+  deleteButton: 'De-authorize this client',
```

Comment:
```suggestion
  deleteButton: 'Revoke authorization',
```
Let's word the messaging around this like this. `De-authorize` is a bit clunky.